modules/API/external/external_api.js

```diff
@@ -224,7 +225,11 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
                 }
             })
         });
-        this._numberOfParticipants = 1;
+        this._isLargeVideoVisible = true;
+        this._numberOfParticipants = 0;
+        this._participants = {};
```

Comment:
I'm wondering if a potential future refactor is to get the number of participants from the _participants object instead of keeping it in a separate state.

Reply:
Totally makes sense!